Dockerfile

```diff
@@ -0,0 +1,38 @@
+FROM debian:jessie
```

Comment:
I'm not sure, but I know resin.io offers a service to put containers on embedded device, so I guess it should work. A quick look at their repos shows that they are building docker for arm (https://github.com/resin-io/docker), but I don't think it is officially supported.